tests/net/gnrc_sixlowpan_frag_sfr_congure_impl/tests/01-run.py

```diff
@@ -130,7 +130,7 @@ def test_fragmentation(factory, zep_dispatch):
         # 2 intermediate hops, 64 - 2
         assert_result(result, 34, 1, 64 - 2)
 
-        result = parser.parse(D.ping6(root_addr, count=100, interval=30, packet_size=500))
+        result = parser.parse(D.ping6(root_addr, count=100, interval=200, packet_size=500))
```

Comment:
Why this change?

Reply:
Sending frames takes now longer (should match the time it takes on the real hardware)